<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBFormDlg</class>
 <widget class="QWidget" name="DBFormDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/resources.qrc">
    <normaloff>:/aplicacion/images/alephLogoIcon.ico</normaloff>:/aplicacion/images/alephLogoIcon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbButtons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbNew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Crea un nuevo registro (F2)</string>
        </property>
        <property name="statusTip">
         <string>Crea un nuevo registro (F2)</string>
        </property>
        <property name="whatsThis">
         <string>Permite crear un nuevo tipo de registro en esta tabla.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/edit_add.png</normaloff>:/generales/images/edit_add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Modificar el registro seleccionado (F3)</string>
        </property>
        <property name="statusTip">
         <string>Modificar el registro seleccionado (F3)</string>
        </property>
        <property name="whatsThis">
         <string>Abre el formulario de edición del registro actualmente seleccionado.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/edit_edit.png</normaloff>:/generales/images/edit_edit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCopy">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Copia los datos del registro actual a uno nuevo (F4)</string>
        </property>
        <property name="statusTip">
         <string>Copia los datos del registro actual a uno nuevo (F4)</string>
        </property>
        <property name="whatsThis">
         <string>Utilice esta opción si desea trabajar con un nuevo registro pero partiendo de los datos de un registro ya existente.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/copyrecord.png</normaloff>:/generales/images/copyrecord.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDelete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Borrar registro seleccionado (F5)</string>
        </property>
        <property name="statusTip">
         <string>Borrar registro seleccionado (F5)</string>
        </property>
        <property name="whatsThis">
         <string>Borra el registro actualmente seleccionado.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/delete.png</normaloff>:/generales/images/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Buscar... (F6)</string>
        </property>
        <property name="statusTip">
         <string>Buscar... (F6)</string>
        </property>
        <property name="whatsThis">
         <string>Abre el formulario de búsqueda.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/edit_search.png</normaloff>:/generales/images/edit_search.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAdjust">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ajustar visualización</string>
        </property>
        <property name="statusTip">
         <string>Ajustar visualización</string>
        </property>
        <property name="whatsThis">
         <string>Ajusta el ancho de las líneas de la rejilla con los datos, para permitir una mejor visualización de los datos.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/edit_resizelines.png</normaloff>:/generales/images/edit_resizelines.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>624</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbOk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cierra el formulario seleccionando el registro marcado</string>
        </property>
        <property name="statusTip">
         <string>Cierra el formulario seleccionando el registro marcado</string>
        </property>
        <property name="whatsThis">
         <string>Cierra el formulario seleccionando el registro marcado</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cerrar (F10)</string>
        </property>
        <property name="statusTip">
         <string>Cerrar (F10)</string>
        </property>
        <property name="whatsThis">
         <string>Cierra este formulario</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/resources.qrc">
          <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F10</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="DBFilterTableView" name="dbFilterTableView">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/edit_add.png</normaloff>:/generales/images/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Crear Presupuesto</string>
   </property>
   <property name="toolTip">
    <string>Abre el formulario de creación de un presupuesto</string>
   </property>
   <property name="statusTip">
    <string>Abre el formulario de creación de un presupuesto</string>
   </property>
   <property name="whatsThis">
    <string>Abre el formulario de creación de un presupuesto</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/edit_edit.png</normaloff>:/generales/images/edit_edit.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="toolTip">
    <string>Permite editar o modificar un presupuesto existente</string>
   </property>
   <property name="statusTip">
    <string>Permite editar o modificar un presupuesto existente</string>
   </property>
   <property name="whatsThis">
    <string>Permite editar o modificar un presupuesto existente</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/delete.png</normaloff>:/generales/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
   <property name="iconText">
    <string>Borrar el presupuesto seleccionado en la lista inferior</string>
   </property>
   <property name="toolTip">
    <string>Borrar el presupuesto seleccionado en la lista inferior</string>
   </property>
   <property name="statusTip">
    <string>Borrar el presupuesto seleccionado en la lista inferior</string>
   </property>
   <property name="whatsThis">
    <string>Borrar el presupuesto seleccionado en la lista inferior</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/edit_search.png</normaloff>:/generales/images/edit_search.png</iconset>
   </property>
   <property name="text">
    <string>Abre el formulario de búsqueda de presupuestos</string>
   </property>
   <property name="iconText">
    <string>Abre el formulario de búsqueda de presupuestos. Permite realizar filtrados más detallados que los que se obtienen con el filtrado rápido.</string>
   </property>
   <property name="toolTip">
    <string>Abre el formulario de búsqueda de presupuestos. Permite realizar filtrados más detallados que los que se obtienen con el filtrado rápido.</string>
   </property>
   <property name="statusTip">
    <string>Abre el formulario de búsqueda de presupuestos. Permite realizar filtrados más detallados que los que se obtienen con el filtrado rápido.</string>
   </property>
   <property name="whatsThis">
    <string>Abre el formulario de búsqueda de presupuestos. Permite realizar filtrados más detallados que los que se obtienen con el filtrado rápido.</string>
   </property>
  </action>
  <action name="actionAdjustLines">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/edit_resizelines.png</normaloff>:/generales/images/edit_resizelines.png</iconset>
   </property>
   <property name="text">
    <string>Ajusta el ancho de las filas de la lista, para una mejor visualización.</string>
   </property>
   <property name="iconText">
    <string>Ajusta el ancho de las filas de la lista, para una mejor visualización.</string>
   </property>
   <property name="toolTip">
    <string>Ajusta el ancho de las filas de la lista, para una mejor visualización.</string>
   </property>
   <property name="statusTip">
    <string>Ajusta el ancho de las filas de la lista, para una mejor visualización.</string>
   </property>
   <property name="whatsThis">
    <string>Ajusta el ancho de las filas de la lista, para una mejor visualización.</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Sale de la aplicación.</string>
   </property>
   <property name="iconText">
    <string>Sale de la aplicación.</string>
   </property>
   <property name="toolTip">
    <string>Sale de la aplicación.</string>
   </property>
   <property name="statusTip">
    <string>Sale de la aplicación.</string>
   </property>
   <property name="whatsThis">
    <string>Sale de la aplicación.</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/generales/images/copyrecord.png</normaloff>:/generales/images/copyrecord.png</iconset>
   </property>
   <property name="text">
    <string>Agregar Usando Actual</string>
   </property>
   <property name="toolTip">
    <string>Agrega un nuevo presupuesto copiando los datos del presupuesto seleccionado</string>
   </property>
  </action>
  <action name="actionOk">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="toolTip">
    <string>Cierra el formulario seleccionando el registro actualmente marcado</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBFilterTableView</class>
   <extends>QWidget</extends>
   <header>widgets/dbfiltertableview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dbFilterTableView</tabstop>
  <tabstop>pbNew</tabstop>
  <tabstop>pbEdit</tabstop>
  <tabstop>pbCopy</tabstop>
  <tabstop>pbDelete</tabstop>
  <tabstop>pbSearch</tabstop>
  <tabstop>pbAdjust</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
